API.md

```diff
@@ -640,6 +640,44 @@ Returns:
 
 ---
 
+### Respondents enrolled in a survey for a business
+
+* `GET /respondents/survey_id/<survey_id>/business_id/<business_id>`
+  * returns a list of Respondents enrolled in a survey for a specified business id.
+
+### Example JSON Response
+
+```json
+[
+   {
+      "id":"0bdcda6c-5d5c-43bf-9ebd-7085b37f9035",
+      "sampleUnitType":"BI",
+      "emailAddress":"a@z.com",
+      "firstName":"A",
+      "lastName":"Z",
+      "telephone":"123",
+      "status":"CREATED",
+      "associations":[
+         {
+            "partyId":"3b136c4b-7a14-4904-9e01-13364dd7b972",
+            "sampleUnitRef":"49000000000",
+            "businessRespondentStatus":"CREATED",
+            "enrolments":[
+               {
+                  "surveyId":"cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87",
+                  "enrolmentStatus":"ENABLED"
+               }
+            ]
+         }
+      ],
+      "password_verification_token":"ImFAei5jb20i.Zj3biA.PAgq3GiYIdfeUywntZS7EDeKIzY",
+      "password_reset_counter":1
+   }
+]
+```
```

Comment:
This is not quite right, it's a list of objects each containing the `enrolment_status` and a `respondent` object.
```
    {
        "enrolment_status": "DISABLED",
        "respondent": {
            "emailAddress": "a@z.com",
            "firstName": "A",
            "id": "8010aacc-2858-410e-9c65-c2a542d7589e",
            "lastName": "Z",
            "markForDeletion": false,
            "password_reset_counter": 0,
            "password_verification_token": null,
            "pendingEmailAddress": "b@y.com",
            "sampleUnitType": "BI",
            "status": "ACTIVE",
            "telephone": "01234567890"
        }
    }
```